bench/wtperf/config.c

```diff
@@ -653,6 +674,45 @@ config_sanity(CONFIG *cfg)
 }
 
 /*
+ * config_to_file --
+ *	Write the final config used in this execution to a file.
+ */
+void
+config_to_file(CONFIG *cfg)
+{
+	CONFIG_QUEUE_ENTRY *config_line;
+	FILE *fp;
+	size_t req_len;
+	char *path;
+
+	fp = NULL;
+
+	/* Backup the config */
+	req_len = strlen(cfg->home) + 100;
+	if ((path = calloc(req_len, 1)) == NULL) {
+		(void)enomem(cfg);
+		goto err;
+	}
+
+	snprintf(path, req_len + 14, "%s/CONFIG.wtperf", cfg->home);
+	if ((fp = fopen(path, "w")) == NULL) {
+		lprintf(cfg, errno, 0, "%s", path);
+		goto err;
+	}
+
+	/* Print the config dump */
+	while (!TAILQ_EMPTY(&cfg->config_head)) {
+		config_line = TAILQ_FIRST(&cfg->config_head);
+		TAILQ_REMOVE(&cfg->config_head, config_line, c);
+		fprintf(fp, "%s\n", config_line->string);
+	}
+err:
+	free(path);
+	if (fp != NULL)
+		(void)fclose(fp);
+}
+
+/*
  * config_print --
  *	Print out the configuration in verbose mode.
```

Comment:
I don't personally believe that this makes the output obsolete. 

I find the output useful when running tests as I can see the config options and startup params on screen just prior to the test starting.

Updating it seems worthwhile for a second ticket.